hyperspy/_signals/eds_tem.py

```diff
@@ -452,3 +452,30 @@ def decomposition(self,
             navigation_mask=navigation_mask, *args, **kwargs)
         self.learning_results.loadings = np.nan_to_num(
             self.learning_results.loadings)
+
+    def create_model(self, auto_background=True, auto_add_lines=True,
+                     *args, **kwargs):
+        """Create a model for the current TEM EDS data.
+
+        Parameters
+        ----------
+        auto_background : boolean, default True
+            If True, adds automatically a polynomial order 6 to the model, using
+            the edsmodel.add_polynomial_background method.
+        auto_add_lines : boolean, default True
+            If True, automatically add Gaussians for all X-rays generated in the
+            energy range by an element using the edsmodel.add_family_lines
+            method.
+
+        Returns
+        -------
+
+        model : `EDSTEMModel` instance.
+
+        """
```

Comment:
Ah - that identifies a potential slip -- in the create_model method for eels *args, *kwargs are not included... they're here because I based this on the method in hspy.create_model. Do you think we want them (and then your suggested docstring line) or just remove them?